metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/README.md

```diff
@@ -0,0 +1,3 @@
+# bro Parser
+
+This is the bro parser
```

Comment:
The layout mirrors what the nifi nar archetypes produces, in that it creates a separate bundle/nar project from the code project.  I added the assembly ( from metron's method ) as it's own project because it seemed more correct for me since it has to package both things.